test/data/greaselion-data/1/Greaselion.json

```diff
@@ -51,5 +51,14 @@
         "scripts": [
             "scripts/b-com.js"
         ]
+    },
+    {
+        "urls": [
+            "http://messages.example.com/*"
+        ],
+        "scripts": [
+            "scripts/messages-example-com.js"
+        ],
+        "messages": "_locales"
```

Comment:
not a big deal one way or the other, but since this is a known location we could just make this a boolean or do we even need it at all? Can we just check for the existence of this directory? Might simplify things and ensure there's no chance that older clients will have any issues

Reply:
It's there to support multiple locale directories so all of the strings don't get lumped together. I renamed the directory to `messages` in Greaselion to make that clearer (open to other naming suggestions!)